CHANGELOG.md

```diff
@@ -1,3 +1,112 @@
+# 0.0.106 - 2022-03-30
```

Comment:
Never try to predict the date, you'll jinx yourself :p

Reply:
lol